admin-dev/themes/default/template/layout-ajax.tpl

```diff
@@ -79,13 +79,13 @@
 		<div class="alert alert-warning">
 			<button type="button" class="close" data-dismiss="alert">&times;</button>
 			{if count($warnings) > 1}
-				{l s='There are %d warnings.' sprintf=count($warnings)}
+				{l s='There are %d warnings.' sprintf=count($warnings) d='Admin.Actions'}
 				<span style="margin-left:20px;" id="labelSeeMore">
 					<a id="linkSeeMore" href="#" style="text-decoration:underline">{l s='Click here to see more'}</a>
-					<a id="linkHide" href="#" style="text-decoration:underline;display:none">{l s='Hide warning'}</a>
+					<a id="linkHide" href="#" style="text-decoration:underline;display:none">{l s='Hide warning' d='Admin.Actions'}</a>
 				</span>
 			{else}
-				{l s='There is %d warning.' sprintf=count($warnings)}
+				{l s='There is %d warning.' sprintf=count($warnings) d='Admin.Actions'}
```

Comment:
```suggestion
				{l s='There are %d warnings.' sprintf=count($warnings) d='Admin.Notifications.Error'}
```

Reply:
I don't know why this string is not in the plural form like the strings counting errors.

Reply:
À harmoniser au reste de la PR, merci beaucoup Mathieu !